beestation.dme

```diff
@@ -2820,6 +2820,7 @@
 #include "code\modules\modular_computers\computers\machinery\console_presets.dm"
 #include "code\modules\modular_computers\computers\machinery\modular_computer.dm"
 #include "code\modules\modular_computers\computers\machinery\modular_console.dm"
+#include "code\modules\modular_computers\file_system\announcements.dm"
```

Comment:
humble suggestion: don't make PRs, draft or not, where you have done **literally nothing**